src/panels/lovelace/cards/hui-entities-card.ts

```diff
@@ -290,13 +290,18 @@ class HuiEntitiesCard extends LitElement implements LovelaceCard {
   }
 
   private renderEntity(entityConf: LovelaceRowConfig): TemplateResult {
+    let config = entityConf as EntityConfig;
+    if (this._config?.state_color) {
+      config = { state_color: true, ...config } as EntityConfig;
+    }
+    if (this._config?.secondary_info) {
+      config = {
+        secondary_info: this._config!.secondary_info,
+        ...config,
+      } as EntityConfig;
+    }
```

Comment:
Combine these so we dont copy it twice

Reply:
I'm actually not sure how to do that. I tried to make it conditional in the declaration itself, but couldn't get it to work so surrounded it with the outer if-clauses

Reply:
just set it to undefined in a ternary